server/lib/realtime/workflows/trigger.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Realtime.Workflows.Trigger do
```

Comment:
@fracek perhaps we should consider moving the logic in `Trigger.notify` to `Manager` if its only purpose is to get `workflows_for_change` from `Manager` and then invoking workflows.